src/app/components/Book/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+
+import styles from './styles';
+
+interface Props {
+  title: string;
+  author: string;
+  image: string;
+}
+
+const Book = ({ title, author, image }: Props) => {
+  return (
+    <View style={styles.container}>
+      <View style={styles.content}>
+        <Image style={styles.image} source={{ uri: image }} />
```

Comment:
Missing `resizeMode`  for Image => https://reactnative.dev/docs/image#resizemode